albatross/covariance_functions/covariance_functions.h

```diff
@@ -22,6 +22,7 @@
 namespace albatross {
 
 template <typename Term> struct CovarianceFunction {
+  using TermType = Term;
```

Comment:
Is this used? Can we just change the template param name?

Reply:
It's used to be able to access the template parameter from an instantiation of a `CovarianceFunction`.  Ie, to be able to do: `CovarianceFunction::TermType new_term;`